examples/molecular_dynamics/3D/hoomd_periodic.py

```diff
@@ -59,4 +65,4 @@ def simulate(num_particles, dt, tf, profile=False):
 
 if __name__ == '__main__':
     import sys
-    print(simulate(int(sys.argv[1]), 0.02, 4., profile=True))
+    print(simulate(int(sys.argv[1]), 0.02, 200., profile=True, log=True))
```

Comment:
Does turning on profile slow things down?

Reply:
Probably does, but I'm not turning it on in the performance_comparison script